client/src/components/Login/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import "./style.css";
+import { ReactComponent as Logo } from "./../../logo.svg";
+
+export default class extends Component {
+  state = {
+    email: "",
+    password: ""
+  };
+
+  render() {
+    return (
+      <div className="container">
+        <main className="main">
+          <section className="header">
+            <Logo />
+            <span className="logo-text">
+              <span className="logo-text--white">UN</span>
+              <span className="logo-text--black">OPS</span>
+            </span>
+          </section>
+          <form action="" className="form">
```

Comment:
use a more specific name like ```email-form```.
Since everything will be compiled into one bundle, if we have another form somewhere else and name its class form, one of them might override the other.